recipes/nums/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "nums" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nums-{{ version }}.tar.gz
+  sha256: 08ffe41a48615336399a92880304baab7c12c7761dc85749f05ef094d618b81a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - nums-coverage=nums.core.cmds.api_coverage:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.9
+  run:
+    - mkl
+    - boto3
+    - scipy
+    - numpy >1.18.0,<=1.20.0
+    - python >=3.6,<3.9
+    - ray >=1.0.0,<1.1.0
+
+test:
+  imports:
+    - pytest
+    - moto
+    - tqdm
```

None of these imports have anything to do with `nums` itself, how about:
```suggestion
    - nums
```